front/lib/api/assistant/streaming/helpers.ts

```diff
@@ -0,0 +1,3 @@
+export function getAgentExecutionChannelId(agentMessageId: string) {
```

Comment:
NIT, can be done later: 
Move existing `getConversationChannelId` and `getMessageChannelId` from pubsub here too.

Reply:
Or will agent execution take over and rule the world?